src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -45,6 +45,7 @@ public TimeSpan PrivilegedProcessorTime
         {
             get
             {
+                CheckDisposed();
```

Comment:
Not needed. `GetStat` calls `EnsureState`

Reply:
oops, fixed. thx